docs/building/windows-full.md

```diff
@@ -82,10 +76,9 @@ Running Dev version of FullCLR PowerShell
 
 Running FullCLR version is not as simple as CoreCLR version.
 
-If you just run `./powershell.exe`, you will get a `powershell`
-process, but all the interesting DLLs (such as
-`System.Management.Automation.dll`) would be loaded from the Global
-Assembly Cache (GAC), not your output directory.
+If you just run `./powershell.exe`, you will get a `powershell` process, 
+but all the interesting DLLs (such as `System.Management.Automation.dll`) would be loaded 
+from the Global Assembly Cache (GAC), not your output directory.
```

Comment:
by convention, shouldn't at least part of this line (up to the comma) be merged with the line above?